### ISSUE TITLE
B12.1 DOI Service Integration with Registry

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/doi-service#287]
These tasks are to develop the necessary tools to integrate the DOI Service with the Registry.

Some high-level sub-tasks:
* Import DOI service metadata into registry
* Validate DOIs from DOI service match the metadata in the registry from the labels
* Use registry metadata for updating DOI metadata for new version of a product

See sub-tasks for more details.

**Comment by jordanpadams** [comment]
@collinss-jpl after #203 is complete, this will be next up

**Comment by jordanpadams** [comment]
📆 December status: Delayed due to delay on https://github.com/nasa-pds/doi-service/issues/203 . Slack built in. No impact on B12.1 foreseen.

**Comment by tloubrieu-jpl** [comment]
Hi @jordanpadams we will need to discuss this epic at the next breakout.

**Comment by jordanpadams** [comment]
📆 January status: work has started. on schedule

**Comment by jordanpadams** [comment]
📆 February status: Personnel working on DOI service has been pulled off to flight project. Task was a low priority / nice-to-have that will be deferred to a later build as part of a larger metadata integration task

**Comment by jordanpadams** [comment]
Sync with SBN DOIs completed. Follow-on task at future build to integrate with future registry services.
